cypress/support/cypress-support/src/main.js

```diff
@@ -83,6 +84,11 @@ export default function (module) {
       );
     }
     UTILS.destroyGlobalObjects([CONSTANTS.LIFECYCLE_APP_OBJECT_LIST]);
+
+    // Unflatten the openRPC data
+    const flattedOpenRpc = UTILS.getEnvVariable(CONSTANTS.DEREFERENCE_OPENRPC, false);
```

Comment:
UTILS.getEnvVariable(CONSTANTS.DEREFERENCE_OPENRPC, false);I believe it should be true,if is undefined/empty,it's going to throw error in L90.

Reply:
Yes good call! Thanks for pointing that out and I will push that change